### ISSUE TITLE
Document whether debug colors are HSLA or something else

### ISSUE THREAD

**Issue by martin-t** [dimforge/rapier#370]
The [doc comment on DebugColor](https://github.com/dimforge/rapier/blob/master/src/pipeline/debug_render_pipeline/debug_render_style.rs#L5) says that it uses HSLA but [DebugRenderBackend::draw_line](https://github.com/dimforge/rapier/blob/master/src/pipeline/debug_render_pipeline/debug_render_backend.rs#L34) takes `[f32; 4]` so it's not clear from the docs what format the color is meant to be. If rapier uses HSLA everywhere, it should probably use `DebugColor` consistently everywhere.

There's a couple more places in code where i saw `[f32; 4]` (such as `sleep_color_multiplier`).

**Comment by sebcrozet** [comment]
Yes, they are all HSLA by default. I agree that we should use `DebugColor` everywhere.

**Comment by MaxBittker** [comment]
these docs currently use them as rgba:
https://rapier.rs/docs/user_guides/javascript/getting_started_js/
